primedev/wsockproxy/dllmain.cpp

```diff
@@ -1,90 +1,29 @@
 #include "loader.h"
 
-#include <shlwapi.h>
 #include <filesystem>
 
-HINSTANCE hLThis = 0;
-FARPROC p[857];
-HINSTANCE hL = 0;
+FARPROC p[73];
+HMODULE hL = 0;
 
-bool GetExePathWide(wchar_t* dest, DWORD destSize)
-{
-	if (!dest)
-		return NULL;
-	if (destSize < MAX_PATH)
-		return NULL;
-
-	DWORD length = GetModuleFileNameW(NULL, dest, destSize);
-	return length && PathRemoveFileSpecW(dest);
-}
-
-wchar_t exePath[4096];
-wchar_t buffer1[8192];
-wchar_t buffer2[12288];
+static wchar_t buffer[4096];
```

Comment:
I'd prefer if this name was a bit better, not a huge deal since I know that's not rly the point of the PR and the variable existed already. Feel free to just mark as resolved if you want

Reply:
okay, what name do you propose?
How about `wsock_path`?

Reply:
Sure (but camel case, so `wsockPath`)